pkg/controller/argocd/policy_rule.go

```diff
@@ -0,0 +1,341 @@
+package argocd
```

Comment:
Looks like this file is a exact duplicate of policyrule.go , can we remove one ?